pyintegration/deephaven/csv.py

```diff
@@ -66,6 +75,11 @@ def read(
 
         csv_specs = (
             csv_specs_builder.hasHeaderRow(not headless)
+            .skipRows(skip_rows)
+            .numRows(num_rows)
+            .ignoreEmptyLines(ignore_empty_lines)
+            .allowMissingColumns(allow_missing_columns)
+            .ignoreExcessColumns(ignore_excess_columns)
             .delimiter(ord(delimiter))
             .quote(ord(quote))
```

Comment:
I am not sure if I like the design, the number of params has increased to 12, what if we need to add more options in the future, the signature of this method might become unwieldy pretty fast. It almost feels like that we should wrap CsvSpec and support building it through a builder pattern to make the code cleaner and more extensible. I think we also need @chipkent's input on this.

Reply:
As long as all the named arguments have the correct defaults, I think it's "ok" to have a lot of them? IMO the builder pattern is useful in java because we don't have named argument support.

Reply:
OK, I am convinced because the Pandas version has even more params